src/pybind/mgr/rook/ci/scripts/bootstrap-rook-cluster.sh

```diff
@@ -27,14 +30,14 @@ setup_minikube_env() {
     fi
 
     rm -rf ~/.minikube
-    minikube start --memory="4096" --cpus="2" --disk-size=10g --extra-disks=1 --driver kvm2
+    minikube start --disk-size=20g --extra-disks=4 --driver kvm2
```

Comment:
I feel like it could be good to continue setting memory to at least 4096 (gigs). IIRC, 2GB is the default, which is low. I have encountered some general stability issues when using that little memory. 

If this is working consistently for these tests, then it seems fine, but if you start seeing issues where the Rook cluster is extra slow or somewhat unstable, consider raising this as a first attempt to fix the issue.

I usually give minimum 6g to my nodes, but I often need to run file, object, and NFS all at the same time.

Reply:
@BlaineEXE thanks for the recommendation. I'll raise it to 6g based on your feedback as these e2e testing could be extended in the future to add more areas such rgw, filesystem, etc.